visualization/pg_joins.js

```diff
@@ -0,0 +1,197 @@
+const { Pool, types }= require('pg')
+const format = require('pg-format');
+
+const pool = new Pool({
+  user: 'bety',
+  database: 'bety',
+  password: '',
+  host: 'localhost',
+  port: 9000,
+});
+
+const fetch_Id= async (client, value,table)=>{
+  try{
+    const query=format('SELECT id FROM %L WHERE name = %s',table,value)
+    const res= (await client.query(query)).rows[0]["id"];
+    return res
+  }
+  catch(e){
+    console.log(e);
+  }
+}
+
+const fetch_cultivarsId= async (client, name, specie_id)=>{
+  try{
+    console.log(name, specie_id)
+    const query=format('SELECT id FROM cultivars WHERE name = %L AND specie_id = %s',name, parseInt(specie_id))
+    const res= (await client.query(query));
+    return res.rows[0]["id"]
+  }
+  catch(e){
+    console.log(e);
+  }
+}
+
+const fetch_sitesId= async (client, sitename)=>{
+  try{
+    console.log(sitename)
+    const query=format('SELECT id FROM sites where sitename = %L',sitename)
+    const res= (await client.query(query));
+    return res.rows[0]["id"]
+  }
+  catch(e){
+    console.log(e);
+  }
+}
+
+const fetch_specieId= async (client, specie)=>{
+  try{
+    const res= (await client.query('SELECT id FROM species WHERE scientificname = $1', [specie]));
+    return res.rows[0]["id"]
+  }
+  catch(e){
+    console.log(e);
+  }
+}
+
+
+const getTable1=async (cultivars, sites_cultivars, sites) => {
```

Comment:
Use a more descriptive name here instead of "getTable1". Perhaps insertCultivarSites?

Reply:
Thanks! for pointing out, will change it in a future commit.